nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -63,17 +128,13 @@ def setup():
                                     affine_to_rasmm=np.eye(4))
 
     DATA['streamlines_func'] = lambda: (e for e in DATA['streamlines'])
-    fa_func = lambda: (e for e in DATA['fa'])
-    colors_func = lambda: (e for e in DATA['colors'])
-    mean_curvature_func = lambda: (e for e in DATA['mean_curvature'])
-    mean_torsion_func = lambda: (e for e in DATA['mean_torsion'])
-    mean_colors_func = lambda: (e for e in DATA['mean_colors'])
-
-    DATA['data_per_point_func'] = {'colors': colors_func,
-                                   'fa': fa_func}
-    DATA['data_per_streamline_func'] = {'mean_curvature': mean_curvature_func,
-                                        'mean_torsion': mean_torsion_func,
-                                        'mean_colors': mean_colors_func}
+    DATA['data_per_point_func'] = {
+        'colors': lambda: (e for e in DATA['colors']),
```

Comment:
I found the code cleaner when I break the line this way compared to in the middle of the generator comprehension.